.github/workflows/release.yml

```diff
@@ -27,6 +27,12 @@ jobs:
             toolchain: nightly
             override: true
 
```

Comment:
are macos-latest, ubuntu-latest only amd64? since we do not cross-compile for many archs in make-build, but we just expect to be on the right platform/arch  (ie. ARCH=$(uname -m)) we will not make a binary that works on new Apple M1 or raspberry pi 4 etc..

Reply:
Since we do not have CGO, could make sense to cross-compile ourself for all platforms/archs available?